admin/packaging.py

```diff
@@ -921,6 +932,9 @@ def omnibus_package_builder(
                     # SystemD configuration
                     package_files.child('systemd'):
                         FilePath('/usr/lib/systemd/system'),
+                    # Upstart configuration
```

Comment:
:question: Do we want to install upstart stuff in RPM packages? I can see doing systemd in debs cause that's where Ubuntu is moving towards. Dropping in a few extra files isn't a big deal though.